docs/reference/transform/overview.asciidoc

```diff
@@ -62,3 +62,20 @@ image::images/pivot-preview.jpg["Example of a {transform} pivot in {kib}"]
 IMPORTANT: The {transform} leaves your source index intact. It
 creates a new index that is dedicated to the transformed data.
 
+
+[[transform-performance]]
+==== Performance considerations
```

Comment:
I think we could make a case for putting this section in many places (e.g. Troubleshooting, Limitations, or in a whole new page).  I kind of expected it to be a new page, but given how short it is, I think this location makes sense.  If it grows later, I think we should reconsider.

Reply:
@lcawl Exactly my thoughts. I planned to add as an individual page, but it is very short for that. When we expand the use case, we move it to a new page.